tools/syz-declextract/clangtool/declextract.cpp

```diff
@@ -281,7 +281,13 @@ FieldType Extractor::extractRecord(QualType QT, const RecordType* Typ, const std
       IsAnonymous = true;
     }
     FieldType FieldType = genType(F->getType(), BackupFieldName);
-    int BitWidth = F->isBitField() ? F->getBitWidthValue(*Context) : 0;
+    int BitWidth = F->isBitField()
+#if LLVM_VERSION_MAJOR >= 21
```

Comment:
Thanks for the fix!

I think as we go forward, we would like to support a single reasonably up-to-date llvm commit.
Other interface incompatibilities may require more pervasive changes, or worse some higher level
structural changes.
All possible llvm revisions won't be supported anyway (too old and too new revisions may always not work).
Additionally, all revisions we support are unlikely to be well tested. So it may be lots of effort w/o actual result.

If you tested it with a newer llvm revision, please fix the code for the new revision and leave the revision in:
tools/syz-declextract/README.md